emmet-builders/emmet/builders/mobility_screening/migration_graph.py

```diff
@@ -0,0 +1,64 @@
+class MigrationGraphBuilder(MapBuilder):
+    def __init__(
+        self,
+        materials: MongoStore,
+        electrodes: MongoStore,
+        tasks: MongoStore,
+        target: MongoStore,
```

Comment:
The convention in emmet is to give the target store a specific descriptive label so it is clear what kind of collection is created. Here it may be appropriate to rename this variable to something like mgraph or migration_graph.